### PR TITLE
abduco-0.3

### PR DESCRIPTION
**Pull request by hakujin**
Update abduco to 0.3 and bring the formula in line with dvtm.

### CODE REVIEW

File: Library/Formula/abduco.rb [hunk 1]
```diff
@@ -13,21 +13,14 @@ class Abduco < Formula
...
   test do
-    session = "0.1-homebrew-test-session"
-    expected = "[?25h[999Habduco: #{session}: session terminated with exit status 0\n"
-    output = `printf "
" | #{bin}/abduco -e "
" -c #{session} true &> /dev/null && printf "\n" | #{bin}/abduco -a #{session} 2>&1 | sed '$d' | tail -1 | sed 's/.$//'`
-    assert_equal expected, output
+    result = shell_output("#{bin}/abduco -v")
+    result.force_encoding("UTF-8") if result.respond_to?(:force_encoding)
+    assert_match /^abduco-[0-9.]+ © 2013-\d{4} Marc André Tanner$/, result
```
Comment (mikemcquaid) [inline Library/Formula/abduco.rb]: Let's trim this string down; the lack of US-ASCII is calling issues.

Comment (mikemcquaid) [inline Library/Formula/abduco.rb]: See http://bot.brew.sh/job/Homebrew%20Pull%20Requests/21404/version=yosemite/console

**Comment by hakujin** [comment]
Okay will you walk me through this Ruby/Jenkins unicode bullshit? How should the regex be written to properly work here?

File: Library/Formula/abduco.rb [hunk 2]
```diff
@@ -13,21 +13,14 @@ class Abduco < Formula
...
   test do
-    session = "0.1-homebrew-test-session"
-    expected = "[?25h[999Habduco: #{session}: session terminated with exit status 0\n"
-    output = `printf "
" | #{bin}/abduco -e "
" -c #{session} true &> /dev/null && printf "\n" | #{bin}/abduco -a #{session} 2>&1 | sed '$d' | tail -1 | sed 's/.$//'`
-    assert_equal expected, output
+    result = shell_output("#{bin}/abduco -v")
+    result.force_encoding("UTF-8") if result.respond_to?(:force_encoding)
+    assert_match /^abduco-[0-9.]+ . 2013-\d{4} Marc Andr. Tanner$/, result
```
Comment (mikemcquaid) [inline Library/Formula/abduco.rb]: Just make this something like `abduco-#{version}` and do a partial match.

**Comment by bfontaine** [comment]
The 0.3 release has been released 3 days ago.

* URL: http://www.brain-dump.org/projects/abduco/abduco-0.3.tar.gz
* SHA1: 175b2c0eaf2a8b7fb044f1454d018dac4ec31293

**Comment by hakujin** [comment]
@bfontaine @mikemcquaid I updated the formula to abduco-0.3, including a fix from upstream.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `abduco 0.3`.

**Pull request closed without merging**